src/main/java/com/zeiterfassung/hwr/desktop/controller/ButtonPaneController.java

```diff
@@ -56,6 +81,13 @@ public void setController()
         setBtnEventHandler(projects);
     }
 
+    /**
+     * this methode is to get the last Project name (optional)
```

Comment:
<s>this methode is to</s> get the _optional_ last Project name <s>(optional)</s>